src/Microsoft.ApplicationInsights.AspNetCore/ApplicationInsightsJavaScript.cs

```diff
@@ -48,9 +29,7 @@ public string FullScript
             {
                 if (!string.IsNullOrEmpty(this.telemetryConfiguration.InstrumentationKey))
                 {
-                    return ScriptPrefix + @"{
-  instrumentationKey: '" + this.telemetryConfiguration.InstrumentationKey + @"'
-}" + ScriptPostfix;
+                    return string.Format(Resources.JavaScriptSnippet, this.telemetryConfiguration.InstrumentationKey);
```

Comment:
will it read from resources for every request or it will cache it in the string?